### PR TITLE
Update maintenance status

### PR DESCRIPTION
**Pull request by sorah**
We've retired our RRRSpec cluster. :wave:

cc @draftcode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ RRRSpec is used in production as a CI service, running 60+ RSpec processes
 concurrently, and it undergoes those failures, which include lots of `rb_bug`s,
 assertion errors, and segmentation faults.
 
+## MAINTENANCE NOTICE
+
+As of Feb 2019, we, Cookpad, are not using RRRSpec internally. We may ocassionaly review and accept PRs, but note that this software is not under active maintainance.
```
Comment (mame) [inline README.md]: nit: occasionally

File: README.md [hunk 2]
```diff
@@ -10,6 +10,12 @@ RRRSpec is used in production as a CI service, running 60+ RSpec processes
...
+
+(Background: The circumstances has been changed since 2013. Modern IaaS providers such as AWS now provides a machine with many vCPUs under the cheap pricing. Also, Cookpad is working to split their largest Rails monolith application ([blog post :jp:](https://techlife.cookpad.com/entry/2018-odaiba-strategy)). Thus, we decided to stop operating RRRSpec cluster.)
```
Comment (mame) [inline README.md]: nit: provides -> provide

**Review by mame — COMMENTED** [review]
nitpicking

File: README.md (hunk 2 above)
Comment (osyoyu) [reply README.md]: the cheap pricing -> cheap pricing (remove the the)

Comment (osyoyu) [reply README.md]: a machine -> machines

**Comment by yasaichi** [comment]
@sorah 
Would you release the changes from v0.4.4 in future? because I'd like to apply [this patch](https://github.com/cookpad/rrrspec/pull/66) to my application.

P.S. Thanks for your development of these awesome gems until now!

**Comment by sorah** [comment]
@yasaichi Oops, I've forgot about that. Pushed v0.4.5 just now

**Pull request merged**